### PR TITLE
[WebDriver BiDi] `browsingContext.traverseHistory` only for top-level navigables

### PR DESCRIPTION
**Pull request by sadym-chromium**
Align with spec changes introduced in https://github.com/w3c/webdriver-bidi/pull/770

### CODE REVIEW

File: webdriver/tests/bidi/browsing_context/traverse_history/invalid.py [hunk 1]
```diff
@@ -42,3 +42,24 @@ async def test_delta_invalid_value(bidi_session, current_url, new_tab, inline, v
         await bidi_session.browsing_context.traverse_history(
             context=new_tab["context"], delta=value
         )
+
+
+@pytest.mark.parametrize("domain", ["", "alt"],
+                         ids=["same_origin", "cross_origin"])
```
Comment (whimboo) [inline webdriver/tests/bidi/browsing_context/traverse_history/invalid.py:48]: We no longer need the parameterization given that it doesn't matter what kind this iframe will be.

**Review by whimboo — CHANGES_REQUESTED** [review]
Thanks! Overall looks fine. Just one thing to update.

File: webdriver/tests/bidi/browsing_context/traverse_history/invalid.py (hunk 1 above)
Comment (sadym-chromium) [reply webdriver/tests/bidi/browsing_context/traverse_history/invalid.py]: done

**Review by whimboo — APPROVED** [review]
Thanks!

**Pull request merged**